fenixedu-ist-giaf-contracts/src/main/java/pt/ist/fenixedu/contracts/domain/accessControl/DepartmentPresidentStrategy.java

```diff
@@ -86,8 +86,29 @@ public static Person getCurrentDepartmentPresident(Department department) {
         return null;
     }
 
+    public static Set<Person> getCurrentDepartmentPresidents(Department department) {
```

Comment:
This should not be public.
Why create a new public API if the method is ever only used locally?